authority/provisioner/acme.go

```diff
@@ -216,10 +226,50 @@ func (p *ACME) Init(config Config) (err error) {
 		return fmt.Errorf("failed initializing Wire options: %w", err)
 	}
 
+	if slices.Contains(p.AttestationFormats, "android-key") && len(p.RootCRLs) == 0 {
+		p.initializeAndroidCRL()
+	}
+
 	p.ctl, err = NewController(p, p.Claims, config, p.Options)
 	return
 }
 
+const ANDROID_ATTESTATION_STATUS_URL = "https://android.googleapis.com/attestation/status"
+
+// fetch CRL https://android.googleapis.com/attestation/status and build a list of serial number
+func (p *ACME) initializeAndroidCRL() error {
+	log.Printf("Updating Android CRL list for %s ACME provisioner", p.Name)
+	var CRLResponse struct {
+		Entries map[string]struct {
+			Status string `json:"status"`
+			Reason string `json:"reason"`
+		} `json:"entries"`
+	}
+	res, err := http.Get(ANDROID_ATTESTATION_STATUS_URL)
+	if err != nil {
+		return fmt.Errorf("client: error making Android CRL request: %s\n", err)
+	}
```

Comment:
Can you change this into using the `http.Client` that's available on the `Controller`?